doofinder/includes/settings/class-accessors.php

```diff
@@ -2,355 +2,356 @@
+     *
+     * Just an alias for "get_option", because ideally we don't
+     * want to replace the option name in multiple files.
+     *
+     * @param string $language Language code.
+     *
+     * @return bool
+     */
+    public static function is_js_layer_enabled($language = '')
+    {
+        return (bool) get_option(self::option_name_for_language(
+            'doofinder_for_wp_enable_js_layer',
+            $language
+        ));
+    }
+
+    /**
+     * Enable JS Layer.
+     *
+     * Just an alias for "update_option", because ideally we don't
+     * want to replace the option name in multiple files.
+     *
+     * @param string $language Language code.
+     */
+    public static function enable_js_layer($language = '')
+    {
+        update_option(self::option_name_for_language(
+            'doofinder_for_wp_enable_js_layer',
+            $language
+        ), true);
+    }
+
+    /**
+     * Disable JS Layer.
+     *
+     * Just an alias for "update_option", because ideally we don't
+     * want to replace the option name in multiple files.
+     *
+     * @param string $language Language code.
+     */
+    public static function disable_js_layer($language = '')
+    {
+        update_option(self::option_name_for_language(
+            'doofinder_for_wp_enable_js_layer',
+            $language
+        ), false);
+    }
+
+    /**
+     * Retrieve the code of the JS Layer.
+     *
+     * Just an alias for "get_option", because ideally we don't
+     * want to replace the option name in multiple files.
+     *
+     * @param string $language Language code.
+     *
+     * @return string
+     */
+    public static function get_js_layer($language = '')
+    {
+        return wp_unslash(get_option(self::option_name_for_language(
+            'doofinder_for_wp_js_layer',
+            $language
+        )));
+    }
+
+    /**
+     * Update the value of the JS Layer script.
+     *
+     * Just an alias for "update_option", because ideally we don't
+     * want to replace the option name in multiple files.
+     *
+     * @param string $value
+     * @param string $language Language code.
+     */
+    public static function set_js_layer($value, $language = '')
+    {
+        update_option(self::option_name_for_language('doofinder_for_wp_js_layer', $language), $value);
+    }
+
+    /**
+     * Generate the name of the option for a given language.
+     *
+     * Values of the fields for different languages are stored under different options.
+     * Language code is added to option name, except for default language, because we want
+     * settings for default language be exactly the same as if language plugin
+     * was disabled.
+     *
+     * @param string $option_name Base option name, before adding a suffix.
+     * @param string $language Language code.
+     *
+     * @return string Option name with optionally added suffix.
+     */
+    private static function option_name_for_language($option_name, $language = '')
```

Comment:
private static

Reply:
What is the problem with private static?